tests/kuttl/common/assert_sample_deployment.yaml

```diff
@@ -115,8 +115,8 @@ spec:
           successThreshold: 1
           timeoutSeconds: 5
         resources: {}
-      serviceAccount: ovn-operator-ovn
-      serviceAccountName: ovn-operator-ovn
+      serviceAccount: ovnnorth-ovnnorthd-sample
+      serviceAccountName: ovnnorth-ovnnorthd-sample
```

Comment:
If we fix the naming thing I mentioned above, this will need to change too I think.